pkg/kf/commands/apps/push.go

```diff
@@ -235,13 +237,31 @@ func NewPushCommand(p *config.KfParams, client apps.Client, pusher apps.Pusher,
 					pushOpts = append(pushOpts, apps.WithPushContainerImage(app.Docker.Image))
 				}
 
+				// Bind service if set
+				for _, serviceInstance := range app.Services {
+
+					binding, created, err := serviceBindingClient.GetOrCreate(
+						serviceInstance,
+						app.Name,
+						servicebindings.WithCreateBindingName(serviceInstance),
+						servicebindings.WithCreateNamespace(p.Namespace))
+					if err != nil {
+						return err
+					}
+					if created {
+						output.WriteBindingDetails(cmd.OutOrStdout(), binding)
```

Comment:
WDYT about using a `describe.SectionWriter` to produce something like this to give users more context about what's happening during this part of their push:

```
Binding Services:
  user-db:        created
  redis-cache-02: already exists
  custom-svc:     already exists
```

Actually, after discussing this it looks like this'll go away once it's done server-side.